src/components/diagrams/networkAreaDiagram/network-area-diagram-content.jsx

```diff
@@ -33,6 +38,75 @@ function NetworkAreaDiagramContent(props) {
      * DIAGRAM CONTENT BUILDING
      */
 
+    const dynamicCssRules = useMemo(
```

Comment:
no need to use "useMemo", create this out of the component since it does not depend on any react state
you will be able to remove it from useLayoutEffect then

Reply:
Yes, thanks !